src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  const longTermThreshold = 7;
+  const longTermDiscount = 50;
+  const shortTermThreshold = 3;
+  const shortTermDiscount = 20;
+
+  const totalBaseCost = days * dailyRate;
```

Comment:
Consider using `const` for `totalBaseCost` instead of `let` since its value is not being reassigned. This ensures that the variable remains immutable and prevents accidental changes to its value.